htdocs/comm/propal/list.php

```diff
@@ -635,16 +635,29 @@
 if ($search_availability) {
 	$sql .= " AND p.fk_availability IN (".$db->sanitize($db->escape($search_availability)).')';
 }
+$societe_add_ref_in_list = (!empty($conf->global->SOCIETE_ADD_REF_IN_LIST) ? $conf->global->SOCIETE_ADD_REF_IN_LIST : 0); // Récupérer la valeur de SOCIETE_ADD_REF_IN_LIST
```

Comment:
Can you replace conf->global->xxx with getDolGlobalString('xxx
Old syntax is no more allowed.

Reply:
It's ok now